pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
```

Comment:
Why do we want to rebuild the build container every time? Can we not just make the one we have public and add a separate trigger to build it whenever the `build/` directory changes?

Reply:
+1 on the confusion of having these two containers in the same build file. They are for different purpose and are built on different cadence, right?

Reply:
The build provenance team needs a single cloud build file that does everything. If we separate these two files, they will need to build two distinct builds, and pipeline them across each other. The only negative that I know of, is that adding this step will add about 1 minute to the build step, the entire build takes 5 minutes.

Reply:
My preference would be to have two builds, but in the grand scheme of things it's not a big deal. It would be nice to have an automatically generated build container image that is always up to date...